kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -82,7 +86,7 @@
           <!-- Logic for role tags -->
           <td class="table-cell table-role">
             <span v-if="user.kind !== LEARNER" class="user-role">
-              {{ user.kind === ADMIN ? $tr('admin') : $tr('coach') }}
+              {{ user.kind === ADMIN ? $tr('admin', {count: 1}) : $tr('coach') }}
```

Comment:
I'm confused why `count` is getting passed here - it doesn't seem to be used in the `$tr` string